src/vellum/prompts/blocks/autocasting.py

```diff
@@ -0,0 +1,31 @@
+from vellum import ChatMessagePromptBlock, PlainTextPromptBlock, RichTextPromptBlock
```

Comment:
Instead of `autocasting` rename this file to a single word that is similar to "helpers" or "higher order functions" or something like that. just one word though